.azure-pipelines.yml

```diff
@@ -101,60 +115,78 @@ stages:
   - template: build/stage-build.yml
     parameters:
       applicationEnvironment: Staging
+      androidKeyStoreFile: $(InternalKeystore)
+      androidVariableGroup: 'ApplicationTemplate.Distribution.Internal.Android'
+      iosProvisioningProfileFile: $(InternalProvisioningProfile)
+      iosCertificateFile: $(InternalCertificate)
+      iosVariableGroup: 'ApplicationTemplate.Distribution.Internal.iOS'
 
 - stage: Build_Production
   dependsOn: Build_Staging # Use dependsOn: [] if you want to run both builds in parallel.
-  condition: and(succeeded(), eq(variables['IsReleaseBuild'], 'true'))
+  condition: and(succeeded(), eq(variables['IsLightBuild'], 'false'))
   jobs:
   - template: build/stage-build.yml
     parameters:
       applicationEnvironment: Production
+      androidKeyStoreFile: $(GooglePlayKeystore)
+      androidVariableGroup: 'ApplicationTemplate.Distribution.GooglePlay'
+      iosProvisioningProfileFile: $(AppStoreProvisioningProfile)
+      iosCertificateFile: $(AppStoreCertificate)
+      iosVariableGroup: 'ApplicationTemplate.Distribution.AppStore'
 
 - stage: AppCenter_Staging
-  condition: and(succeeded(), eq(variables['IsReleaseBuild'], 'true'))
+  condition: and(succeeded(), eq(variables['IsLightBuild'], 'false'))
   dependsOn: Build_Staging
 
   jobs:
   - template: build/stage-release-appcenter.yml
     parameters:
+      applicationEnvironment: Staging
+      deploymentEnvironment: AppCenter
       appCenterUWPSlug: $(AppCenterUWPSlug)
       appCenteriOSSlug: $(AppCenteriOSSlug)
       appCenterAndroidSlug: $(AppCenterAndroidSlug)
-      deploymentEnvironment: 'Staging'
+      androidKeyStoreFile: $(InternalKeystore)
+      androidVariableGroup: 'ApplicationTemplate.Distribution.Internal.Android'
 
 - stage: AppCenter_Production
-  condition: and(succeeded(), eq(variables['IsReleaseBuild'], 'true'))
+  condition: and(succeeded(), eq(variables['IsLightBuild'], 'false'))
   dependsOn: Build_Production
 
   jobs:
   - template: build/stage-release-appcenter.yml
     parameters:
+      applicationEnvironment: Production
+      deploymentEnvironment: 'AppCenter Prod'
```

Comment:
We could go the whole way: 'AppCenter Production'

Reply:
I though about it, but the app names in AppCenter are titled "{ProjectName} Prod" so I though it might be easier to associate when reading.